gdcdictionary/schemas/molecular_test.yaml

```diff
@@ -6940,7 +6980,26 @@ properties:
   test_units:
     $ref:
       - "_terms.yaml#/test_units/common"
-    type: string
+    enum:
+      - g/dL
+      - g/L
+      - mcg/mL
+      - mg/dl
+      - mg/L
+      - mmol/L
+      - mU/mol Hb
+      - U/g Hb
+      - U/mL
+      - U/L
+      - ukat/L
+      - umol/L
+      - x10^3 cells/mcL
+      - x10^9 cells/L
+      - Not Report
+      - Unknown
+      - count x10^9 cells/L
+    deprecated_enum:
+      - count x10^9 cells/L
```

Comment:
DICT-95 also lists x10^3/mcL under deprecated

Reply:
mg/dl <- Should this be "mg/dL" to match with the other enums?

Reply:
Also "x10^9/L" is under the deprecated value.

Reply:
changed mg/dl to mg/dL 
x10^3/mcL & x10^9/L were added to ticket by mistake, removed from ticket